### PR TITLE
Manhua Plus - new extension

### PR DESCRIPTION
**Pull request by SnakeDoc83**
Closes https://github.com/inorichi/tachiyomi-extensions/issues/2570

Needs icons.

### CODE REVIEW

**Comment by as280093** [comment]
**Icons:**
[ic_launcher.zip](https://github.com/inorichi/tachiyomi-extensions/files/4433238/ic_launcher.24.zip)

File: src/en/manhuaplus/src/eu/kanade/tachiyomi/extension/en/manhuaplus/ManhuaPlus.kt [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            }
+            date_upload = element.select("div.col-xs-4").text().toDate()
+        }
+    }
+
+    private val dateFormat = SimpleDateFormat("HH:mm - dd/MM/yyyy Z", Locale.US)
+
+    private fun String?.toDate(): Long {
+        return try {
+            dateFormat.parse("$this +0800").time
```
Comment (arkon) [inline src/en/manhuaplus/src/eu/kanade/tachiyomi/extension/en/manhuaplus/ManhuaPlus.kt:120]: Why this particular timezone?

Comment (SnakeDoc83) [reply src/en/manhuaplus/src/eu/kanade/tachiyomi/extension/en/manhuaplus/ManhuaPlus.kt:120]: Yeah, guess I didn't think that through.

Comment (SnakeDoc83) [reply src/en/manhuaplus/src/eu/kanade/tachiyomi/extension/en/manhuaplus/ManhuaPlus.kt]: Changed to +0500.  The site's times seem to be 9 hours ahead of me and I'm currently GMT -4; so I think that's right.

**Comment by SnakeDoc83** [comment]
Apparently another request (https://github.com/inorichi/tachiyomi-extensions/issues/2580) uses the same template.  Guess I should make this a factory extension.

**Comment by arkon** [comment]
I guess I'll wait for that?

**Comment by SnakeDoc83** [comment]
Yeah, sorry about that.  Finished incorporating the second source; so done tinkering unless you need me to change anything.

**Comment by arkon** [comment]
@as280093 New icons please, whenever you're free.

**Comment by as280093** [comment]
**Here the icons:**
[ic_launcher.zip](https://github.com/inorichi/tachiyomi-extensions/files/4454548/ic_launcher.26.zip)

**Comment by SnakeDoc83** [comment]
Changed icons and added mangasum.com from #2622

**Pull request merged**